tools/ControllerFactory.php

```diff
@@ -99,14 +103,14 @@ public function getPost(): array {
 		return $this->post;
 	}
 
-
-	public function addData($key, $value) {
-		$this->data[$key] = $value;
-	}
-
-	public function render($twigPath, $data) {
-		echo $this->twig->rrender($twigPath, $data);
+	/**
+	 * Affiche la page demandé avec Twig.
+	 *
+	 * @param string $twigPath
+	 * @param array $data
+	 */
+	public function render(string $twigPath, array $data): void {
+		echo $this->twig->render($twigPath, $data);
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6074596)